src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -316,14 +316,56 @@ static void uct_cuda_copy_event_desc_cleanup(ucs_mpool_t *mp, void *obj)
     }
 }
 
+static double uct_cuda_copy_iface_get_device_bw()
+{
+    CUdevice cu_device;
+    int major_version;
+    ucs_status_t status;
+
+    status = UCT_CUDA_FUNC_LOG_ERR(cuCtxGetDevice(&cu_device));
+    if (status != UCS_OK) {
+        /* did not find a context with the calling thread, try device 0 */
+        status = UCT_CUDADRV_FUNC_LOG_ERR(cuDeviceGet(&cu_device, 0));
+        if (status != UCS_OK) {
+            return 0;
+        }
+    }
+
+    status = UCT_CUDADRV_FUNC_LOG_ERR(
+            cuDeviceGetAttribute(&major_version,
+                                 CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR,
+                                 cu_device));
+    if (status != UCS_OK) {
+        return 0;
```

Comment:
do we really want to return bandwidth as 0??
it should be some reasonable default